dokka-integration-tests/maven/build.gradle.kts

```diff
@@ -6,13 +6,38 @@ import dokkabuild.tasks.GitCheckoutTask
 plugins {
     id("dokkabuild.test-integration")
     id("dokkabuild.setup-maven-cli")
+    id("dev.adamko.dev-publish")
 }
 
 dependencies {
     implementation(projects.utilities)
 
     implementation(kotlin("test-junit5"))
     implementation(libs.junit.jupiterApi)
+
+    val dokkaVersion = project.version.toString()
+    // We're using Gradle included-builds and dependency substitution, so we
+    // need to use the Gradle project name, *not* the published Maven artifact-id
+    devPublication("org.jetbrains.dokka:plugin-all-modules-page:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:analysis-kotlin-api:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:analysis-kotlin-descriptors:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:analysis-kotlin-symbols:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:analysis-markdown-jb:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-android-documentation:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-base:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-base-test-utils:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:dokka-core:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-gfm:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-gfm-template-processing:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-javadoc:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-jekyll:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-jekyll-template-processing:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-kotlin-as-java:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-mathjax:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-templating:$dokkaVersion")
+    devPublication("org.jetbrains.dokka:plugin-versioning:$dokkaVersion")
```

Comment:
What would happen if we introduce a new project (module) and forget to add it as a dependency here?